frontend/src/components/Header.tsx

```diff
@@ -8,13 +8,20 @@ import { useRouter } from 'next/router'
 
 import ApplicationNameBar from './ApplicationNameBar'
 import Banner from './Banner'
+import Breadcrumb from './Breadcrumb'
+
+type BreadcrumbItemType = {
```

Comment:
This should be imported from the breadcrumb component and then re-exported here.

Reply:
After some additional thought.. ignore this comment. I think that `BreadcrumbItemType` should be exported from `Breadcrumb.tsx` and then used wherever necessary.